src/test_workflow/integ_test/distribution_tar.py

```diff
@@ -33,7 +33,7 @@ def install(self, bundle_name: str) -> None:
     @property
     def start_cmd(self) -> str:
         start_cmd_map = {
-            "opensearch": "./opensearch-tar-install.sh",
+            "opensearch": "export OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! && ./opensearch-tar-install.sh",
```

Comment:
FYI, the `export OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!` would be redundant for versions below 2.12.0, but should not block the installation.